### PR TITLE
Add sanity-amplitude-experiment plugin

### PR DESCRIPTION
**Pull request by thomasKn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,5 @@
...
-
-Learn more about the power of Turborepo:
-
-- [Tasks](https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks)
-- [Caching](https://turbo.build/repo/docs/core-concepts/caching)
-- [Remote Caching](https://turbo.build/repo/docs/core-concepts/remote-caching)
-- [Filtering](https://turbo.build/repo/docs/core-concepts/monorepos/filtering)
-- [Configuration Options](https://turbo.build/repo/docs/reference/configuration)
-- [CLI Usage](https://turbo.build/repo/docs/reference/command-line-reference)
+Check out [@tinloof/sanity-amplitude-experiment](https://github.com/tinloof/sanity-amplitude-experiment/tree/tin-423-create-document-and-publish-the-library/packages/sanity-amplitude-experiment) for more details.
```
Comment (siffogh) [inline README.md:5]: I think we can put the docs in the main README, no?

File: packages/sanity-amplitude-experiment/src/components/ExperimentInput.tsx [hunk 2]
```diff
@@ -25,10 +17,14 @@ export function ExperimentInput(props: ObjectInputProps) {
     }
   }, [secrets, onChange, value])
 
+  if (loading) {
+    return <Spinner />
```
Comment (siffogh) [inline packages/sanity-amplitude-experiment/src/components/ExperimentInput.tsx:21]: Can we mimic the same loading skeleton that Sanity's reference field uses? I think it looks much better.

Comment (thomasKn) [reply packages/sanity-amplitude-experiment/src/components/ExperimentInput.tsx:21]: No more Spinner => [67afbb2](https://github.com/tinloof/sanity-amplitude-experiment/pull/1/commits/67afbb2bc1d82a1b21298b4291c320f2b20ad297)

File: README.md (hunk 1 above)
Comment (thomasKn) [reply README.md:5]: I think that if you do that we won't be able to have the readme in NPM as well, but I'm not sure.

**Pull request merged**